bitmap-plasma/app/build.gradle

```diff
@@ -7,12 +7,10 @@ android {
         applicationId 'com.example.plasma'
         minSdkVersion 14
         targetSdkVersion 25
-        ndk {
-            abiFilters 'armeabi', 'armeabi-v7a', 'arm64-v8a', 'x86'
-        }
+
         externalNativeBuild {
             cmake {
-                arguments '-DANDROID_PLATFORM=android-14', '-DANDROID_TOOLCHAIN=clang'
+                arguments '-DANDROID_PLATFORM=android-14'
```

Comment:
Superfluous?

Reply:
same as before, mainly sometimes version of minSDK exists, but platforms/android-"$minSDKVersion" does not exist, and build would error out. It is good for application maintenance to explicitly set it: avoiding unnecessary future issues.

Reply:
This isn't the case with new NDK versions. There are no gaps.